.github/workflows/release-packages.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
           echo "deb_package_name=ubuntu-20.04-$deb_package_name" >> $GITHUB_OUTPUT
       - name: Get release info
         id: get_release_info
-        uses: bruceadams/get-release@v1
+        uses: bruceadams/get-release@v1.3.2
```

Comment:
This is surprising to me: isn't this what you get by default (assuming 1.3.2 is the most recent version of v1)?

Reply:
Hi Michael, sorry for late response:

This ended up being needed - I'm not entirely sure why though. There appear to be multiple reports in other projects as well, for instance -> https://github.com/actions/setup-node/issues/211

I think what happens is that there's [two levels of tags that need to have a close correspondence for the shorthand version to work](https://github.com/actions/toolkit/blob/main/docs/action-versioning.md). It may be that the author of this action made a mistake and broke that correspondence between the tags, and this is why the version wasn't being found when it was looked up in a registry by GitHub Actions.

But this is me speculating based on the fix, other reports and some documentation - I'm not entirely sure about the root cause.